e2e/suites/public-form.test.ts

```diff
@@ -0,0 +1,81 @@
+import { adminLogin, logout } from '../helpers/login';
+import proxyMock from '../helpers/proxyMock';
+import insertFixtures from '../helpers/insertFixtures';
+import disableTransitions from '../helpers/disableTransitions';
+
+const selectors = {
+  pageContentsInput:
+    '.page-viewer.document-viewer > div > div.tab-content.tab-content-visible > textarea',
```

Comment:
i think we are abandoning this way of using selectors ? any specific reason to use the full path?, i believe there is only 1 or 2 textarea on the page ?

Reply:
Updated it to '.markdownEditor textarea'  thats the best I can do without modifying the Markdown component.
I think all components we write should accept className.